### DISCUSSION TITLE
[Feature] Slideshow for the album

### DISCUSSION THREAD

**Discussion by fedonr** [immich-app/immich#3571] [Feature Request]
### The feature

Can we add a slideshow like feature to the clients like Web and App, where from the Image Viewer where we hit previous or next, we can also have a Play/pause button there which would play all the photos/videos of that album as a slideshow of sorts.

And people can decide the slide duration in management section, if they like 1 to 10 seconds delay between images.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Danie10** [comment]
It seems the slideshow is only available to the owner of an album too? It would be great if, on the app and web, that those it is shared with can also view it as a slideshow.

**Comment by Batwam** [comment]
That would be a good way to turn an old tablet into a digital frame too, especially if it can be combined with smart albums (albums generated automatically based in keyword/person name)
